src/huggingface_hub/keras_mixin.py

```diff
@@ -444,6 +443,7 @@ def _from_pretrained(
         resume_download,
         local_files_only,
         token,
+        config: Optional[Dict[str, Any]] = None,
```

Comment:
If I'm not mistaken, PyTorch's mixin doesn't take a config as attribute there:

https://github.com/huggingface/huggingface_hub/blob/05585741416faf1ca5364da63c450ae56adbfb9b/src/huggingface_hub/hub_mixin.py#L598-L612

Is this a difference that we want to keep or do we want to have the two mixins behave the same?

Reply:
Yes the difference is that `config` is injected to `_from_pretrained` only if the method expects it. In Pytorch mixin we don't necessarily need it (depends on `__init__`) while the Keras mixin expects it (to do `model.config = config` => not very useful but avoid a breaking change).